### PR TITLE
bug-fix/issue-#9660

### PR DESCRIPTION
**Pull request by gnusi**
fix for https://github.com/arangodb/arangodb/issues/9660

### CODE REVIEW

File: tests/js/common/aql/aql-view-arangosearch-cluster.inc [hunk 1]
```diff
@@ -850,7 +850,24 @@
...
+          "LET x = FIRST(FOR y IN UnitTestsCollection FILTER y.c == 0 RETURN DISTINCT y.a) " +
+          "FOR doc IN UnitTestsView SEARCH doc.a == x && doc.c == 0 OPTIONS { waitForSync: true } " +
+          "LET score = BM25(doc, 20.01, 0.01)*0.0001 + 1.0 " + 
+          "COLLECT a = doc.a AGGREGATE maxScore = MAX(score) " +
+          "RETURN { a: a, score: maxScore }"
+        ).toArray();
+
+        assertEqual(result.length, 1);
+        result.forEach(function (res) {
+          assertTrue(res.a === "foo");
```
Comment (jsteemann) [inline tests/js/common/aql/aql-view-arangosearch-cluster.inc]:
```suggestion
          assertEqual(res.a, "foo");
```

File: tests/js/common/aql/aql-view-arangosearch-noncluster.js [hunk 2]
```diff
@@ -888,7 +888,24 @@ function iResearchAqlTestSuite () {
...
+        "LET x = FIRST(FOR y IN UnitTestsCollection FILTER y.c == 0 RETURN DISTINCT y.a) " +
+        "FOR doc IN UnitTestsView SEARCH doc.a == x && doc.c == 0 OPTIONS { waitForSync: true } " +
+        "LET score = BM25(doc, 20.01, 0.01)*0.0001 + 1.0 " + 
+        "COLLECT a = doc.a AGGREGATE maxScore = MAX(score) " +
+        "RETURN { a: a, score: maxScore }"
+      ).toArray();
+
+      assertEqual(result.length, 1);
+      result.forEach(function (res) {
+        assertTrue(res.a === "foo");
```
Comment (jsteemann) [inline tests/js/common/aql/aql-view-arangosearch-noncluster.js]:
```suggestion
        assertEqual(res.a, "foo");
```

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-linux/7338/

**Pull request merged**